### PR TITLE
Towards #56 - Homogenize end-to-end examples to mimic main2.md

### LINKED ISSUE #56: Homogenize end-to-end examples to mimic main2.md
Right now some of the end-to-end examples don't quite follow the main2.md flow. 

Goal of this issue: get them to follow it
- in particular: steps 2 and 3. In particular, measuring the test_nmse at the end of step 3
- step 1 can simply point to main2.md step 1. Step 4 can simply point to main2.md step 4

Each example needs to produce predictions for 1h, 2h, ..., 12h into the future. Not just e.g. 1h into the future, or just 12h.
- Incorrect: code split the data in training and test and then select 12 samples of the test set as 12 hours predictions 

READMEs to cover:
- [x] [Jaime] Simple uses ETH data from Ocean, and linear models. https://github.com/oceanprotocol/predict-eth/commit/09b121786159e7f50a0d07b43410661185d7257a
- [x] [Jaime] Model comparison for direct prediction 1-12 hours
- ~~[JP] Gentle introduction~~
- ~~[JP] Medium level: MA & EMA. 
  - Ensure that predict 1, 2, .., 12h into future. And, don't use samples of the test set as predictions~~

### PR DESCRIPTION
**Pull request by jfdelgad**
Towards #56

Changes proposed in this PR:

- Homogenize Simple uses ETH data from Ocean, and linear models to mimic main2.py

### CODE REVIEW

**Comment by trentmc** [comment]
FYI if you have "fixes #__" in your PR, then github will automatically close the issue when you merge it into main. So I always use "towards #__", which has no magic side effects from github upon merge, and is more compact. I'll update your PR accordingly now.

File: examples/end-to-end-via-raw-ocean-data-and-linear-models.md [hunk 1]
```diff
@@ -37,7 +37,15 @@ dts = to_datetimes(allcex_uts)
 # create a Data Frame with two columns [date,eth-prices] with dates given in intervals of 1-hour
 import pandas as pd
 data = pd.DataFrame({"ds": dts, "y": allcex_vals})
+```
 
+## 3.  Make predictions
+
+### 3.1  Build an AI model
+
+In Python:
```
Comment (trentmc) [inline examples/end-to-end-via-raw-ocean-data-and-linear-models.md:46]: "In Python:" -> "In the same Python console:"

(Often, little things like this really help reduce friction, it's why we do it)

File: examples/end-to-end-via-raw-ocean-data-and-linear-models.md [hunk 2]
```diff
@@ -47,15 +55,30 @@ test_data = data.iloc[-12:,:]
 from prophet import Prophet
 model = Prophet()
 model.fit(train_data)
+```
+
+### 3.2  Run the AI model to make future ETH price predictions
+
+In Python:
```
Comment (trentmc) [inline examples/end-to-end-via-raw-ocean-data-and-linear-models.md:62]: "In Python:" -> "In the same Python console:"

File: examples/end-to-end-via-raw-ocean-data-and-linear-models.md [hunk 3]
```diff
@@ -47,15 +55,30 @@ test_data = data.iloc[-12:,:]
...
+### 3.3 Calculate NMSE
+
+In Python:
 
-# Calculate Normalized Mean Squared Error between predictions and true (test) values    
+```python
 nmse = calc_nmse(test_data.y, pred_vals)
 print(f"NMSE = {nmse}")
+```
+
```
Comment (trentmc) [inline examples/end-to-end-via-raw-ocean-data-and-linear-models.md:78]: To make section 3.3 even more like main2.md, I'd suggest:

----
We use normalized mean-squared error (NMSE) as the accuracy measure.

In the same Python console:

```python
# now, we have predicted and actual values. Let's find error, and plot!
cex_vals = test_data.y
nmse = calc_nmse(cex_vals, pred_vals)
print(f"NMSE = {nmse}")
plot_prices(cex_vals, pred_vals)
````

Keep iterating in step 3 until you're satisfied with accuracy. Then...

**Review by trentmc — CHANGES_REQUESTED** [review]
Getting close! Just a few small suggested tweaks.

**Review by trentmc — APPROVED** [review]
